### PR TITLE
Feature/Jira 5 hero

### PR DESCRIPTION
**Pull request by HumanSockPuppet**
This pull request is about adding a banner for the online portfolio project.

### CODE REVIEW

**Comment by adimapilis** [comment]
on the `img` directory, I think we can create an index file to export the images,
see this answer on stackoverflow.
https://stackoverflow.com/a/49782755
also kindly delete the unused assets

**Comment by adimapilis** [comment]
don't forget to run `npm run format` and `npm run lint`

File: src/components/Banner.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+//import { ArrowRightCircle } from "react-bootstrap-icons";
+import headerImg from "../assets/img/yoshi 3.png";
+import music from '../assets/music.mp3';
+import { useEffect, useRef } from "react";
+import { BsFillPauseFill, BsFillPlayFill } from "react-icons/bs";
+
+export const Banner = () => { 
+  const [value, setValue] = useState(1);
```
Comment (adimapilis) [inline src/components/Banner.js]: is there a reason why we are using number here instead of a boolean?

File: src/components/Banner.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  const buttonText = value %2 ===0 ? <BsFillPauseFill className="music-icon"/> : <BsFillPlayFill className="music-icon"/>;
```
Comment (adimapilis) [inline src/components/Banner.js]: I don't think this should be named as buttonText

File: src/components/Banner.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+//import { ArrowRightCircle } from "react-bootstrap-icons";
```
Comment (adimapilis) [inline src/components/Banner.js]: i think you forget to clean this one up

File: src/components/Banner.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        <span className="tagline">WELCOME TO MY PORTFOLIO</span>
+      </Container>
+      <Container classname="container">
+        <Row className="align-items-center">
+          <Col cs={12} md={6} xl={6}>
+            <img src={headerImg} alt="Header Img" />
+          </Col>
+
+          <Col cs={12} md={6} xl={6}>
+            <h1>{`It's-a-me, BRIX!`} </h1>
```
Comment (adimapilis) [inline src/components/Banner.js]: Is there a reason why we enclose the text with {``}??

File: src/App.css [hunk 5]
```diff
@@ -52,3 +52,93 @@ nav.navbar .navbar-nav .nav-link.navbar-link-projects {
...
+  color: black;
+}
+
+.banner h1 {
+  font-family: dogica;
+  color: black;
+  font-size: 40px;
+}
+
+@font-face {
```
Comment (adimapilis) [inline src/App.css]: i think it is nicer to place these font declaration on top of the css file

File: src/components/Banner.js (hunk 3 above)
Comment (HumanSockPuppet) [reply src/components/Banner.js]: done

File: src/components/Banner.js (hunk 4 above)
Comment (HumanSockPuppet) [reply src/components/Banner.js]: done

File: src/App.css (hunk 5 above)
Comment (HumanSockPuppet) [reply src/App.css]: done

File: src/components/Banner.js (hunk 1 above)
Comment (HumanSockPuppet) [reply src/components/Banner.js]: fixed

**Pull request merged**